tests/testthat/test_estimate-promoter-activity.R

```diff
@@ -0,0 +1,41 @@
+context('Promoter Activity and Gene Expression')
+library(proActiv)
+
+promoterAnnotation <- promoterAnnotation.gencode.v19
+filesTophat <- list.files(system.file('/extdata/tophat2', package = 'proActiv'), full.names = TRUE, pattern = 'sample1|sample2')
```

Comment:
why not using the same data as in the vignette? are these files smaller?

Reply:
the vignette files follow hg38, while the original promoter annotation and sample files follow hg19. i thought it'd be a better idea to test the code against the original output, hence using the original input files. 
to reduce the number of files, it's possible to generate hg38 promoter annotation with the original code, run it with the vignette files, and compare the output to the run with the current code? but that'll cover only STAR junction files.